redaxo/src/addons/structure/pages/index.php

```diff
@@ -76,9 +76,9 @@
 // $KAT->setDebug();
 if (count($structureContext->getMountpoints()) > 0 && 0 == $structureContext->getCategoryId()) {
     $parent_id = implode(',', $structureContext->getMountpoints());
-    $KAT->setQuery('SELECT COUNT(*) as rowCount FROM ' . rex::getTablePrefix() . 'article WHERE id IN (' . $parent_id . ') AND startarticle=1 AND clang_id=' . $structureContext->getClangId());
+    $KAT->setQuery('SELECT COUNT(*) as rowCount FROM ' . rex::getTablePrefix() . 'article WHERE id IN (' . $parent_id . ') AND startarticle=1 AND clang_id=?', [$structureContext->getClangId()]);
```

Comment:
Die parent-id auch?

Reply:
`IN (...)` ist immer so umständlich in prepared statements, da man alle werte einzeln als param übergeben muss ;)
aber wäre wohl besser.

Reply:
Ok,.. ggf. Sollten wir für IN ne separate API schaffen, damit es komfortabler wird (separater PR)

Reply:
Ja, ich bin seit gestern schon am überlegen, wie dazu eine Api am besten aussehen würde.
Ich werde gleich wahrscheinlich mal einen Vorschlag machen (in separatem PR).

Reply:
Wurde in https://github.com/redaxo/redaxo/pull/4225 umgesetzt